tox.ini

```diff
@@ -19,23 +20,9 @@ basepython =
     py35: python3.5
 
 deps =
-    django-modelcluster>=1.1b1
-    django-taggit>=0.17.5
-    django-treebeard>=3.0,<5.0
     django-sendfile==0.3.6
```

Comment:
> I'm not sure why elasticsearch is in the former list - maybe that was an oversight

Nope, I put it there. Most of the ES tests don't require elasticsearch. This is why we have the ``--elasticsearch`` flag